### ISSUE TITLE
Audio issues

### ISSUE THREAD

**Issue by unrealdtc** [adrian-bl-yuga/device-sony-c6603#52]
Hi,

I'm experiencing some audio issues, like audio dropping in various apps, for example DSub or Pocket Casts. Most times, quitting all running apps except the audio app "resolves" the problem, sometimes not. I'm using headphones plugged straight into the audio jack on the XZ, no bluetooth headphones or external DAC. I'm running the latest version at the time of writing (5.1.1).

**Comment by adrian-bl** [comment]
Did you try switching to some other governor or/and forcing the phone to keep at least 2 CPUs online?
I suppose that the phone might simply run out of CPU power (or fails to handle some spikes, resulting in audio issues)

**Comment by unrealdtc** [comment]
I've tried changing governor to Interactive, which didn't improve the stuttering problem. Won't forcing 2 CPU cores online kill battery life?

**Comment by adrian-bl** [comment]
No: The apq8064 CPU's are pretty good at sleeping: you won't notice a difference in normal usage patterns

**Comment by unrealdtc** [comment]
Thank you! Forcing 2 CPU online + interactive governor seem to resolve this problem for me, and no obvious battery drain. It also seems to be a bitter snappier with interactive governor, compared to ondemand governor, so that's good news all round!
